source/manual/data-gov-uk-monitoring.html.md

```diff
@@ -0,0 +1,48 @@
+---
+owner_slack: "#govuk-datagovuk"
+title: Monitoring for data.gov.uk
+section: data.gov.uk
+layout: manual_layout
+parent: "/manual.html"
+last_reviewed_on: 2018-06-22
+review_in: 6 months
+---
+[publish]: apps/datagovuk_find
+[find]: apps/datagovuk_find
+[paas-metric-exporter]: https://reliability-engineering.cloudapps.digital/manuals/set-up-paas-metric-exporter-with-prometheus.html#configure-container-metrics
+[grafana]: https://grafana-paas.cloudapps.digital/d/xonj40imk/data-gov-uk?refresh=1m&orgId=1
+[sentry]: https://sentry.io/govuk/
+[logit-paas]: https://docs.cloud.service.gov.uk/#set-up-the-logit-io-log-management-service
+[logit]: https://logit.io/a/1c6b2316-16e2-4ca5-a3df-ff18631b0e74
+[google-analytics]: https://sites.google.com/a/digital.cabinet-office.gov.uk/gds/information-management/use-online-tools-in-gds/use-google-analytics
+[ckan]: https://data.gov.uk/user?destination=publisher
+
+## Prometheus
+
+Container metrics are scraped using the [PaaS Metric Exporter][paas-metric-exporter] and sent to [Grafana][grafana]. In a nutshell, the `metric-exporter` app exposes a `/metrics` endpoint listing all of the stats for each container.
+
+## Pingdom
+
+Pingdom checks whether `data.gov.uk` exists and emails `data.gov.uk@digital.cabinet-office.gov.uk` with updates. Maintenace of this service forms part of `#govuk-2nd-line`.
+
+## Sentry
+
+[Sentry][sentry] receives application errors and emails `data.gov.uk@digital.cabinet-office.gov.uk` with updates. The Sentry pagesfor each app can be found on the [Find][find] and [Publish][publish] app pages.
```

Comment:
Reference-style links aren't case sensitive so `[Sentry]` would be enough instead of `[Sentry][sentry]`.

Reply:
Done - thanks :-)